app/models/user.rb

```diff
@@ -117,6 +116,20 @@ def has_facebook_keys?
     facebook_app_id.present? && facebook_app_secret.present?
   end
 
+  def self.role_label(value)
```

Comment:
Please move this code to the users_helper

Reply:
Done